acsoo/templates/project/+project.name+/.bumpversion.cfg.bob

```diff
@@ -0,0 +1,11 @@
+[bumpversion]
+current_version = 1.0.0
+commit = False
+
+[bumpversion:file:.bumpversion.cfg]
+
+[bumpversion:file:acsoo.cfg]
+
+[bumpversion:file:odoo_addons/{{{ project.trigram }}}_all/__openerp__.py]
+
+[bumpversion:file:odoo/addons/{{{ project.trigram }}}_all/__manifest__.py]
```

Comment:
@ThomasBinsfeld `.bumpversion.cfg.bob` must be updated too with the list of `.conf` files, right?

Also put a ```{{% if odoo.series in ['9.0', '8.0'] %}}``` to select `odoo_addons` vs `odoo/addons`.